src/ru/nsu/ccfit/zuev/osu/menu/SongMenu.java

```diff
@@ -1287,6 +1360,79 @@ public boolean checkBG(String filePath) {
     }
 
     public enum SortOrder {
-        Title, Artist, Creator, Date
+        Title, Artist, Creator, Date, Bpm, Stars, Length
+    }
+
+    public enum GroupType {
+        MapSet, SingleDiff
+    }
+
+    private void tryReloadMenuItems(SortOrder order){
+        switch(order){
```

Comment:
This switch statement can be shortened into this:
```java
switch(order){
    case Title:
    case Artist:
    case Creator:
    case Date:
    case Bpm:
        reloadMenuItems(GroupType.MapSet);
        break;
    case Stars:
    case Length:
        reloadMenuItems(GroupType.SingleDiff);
        break;
}
```
Fallthrough doesn't matter in this switch statement. The final result is the same.

Reply:
fixed